### PR TITLE
Update to material 0.10.1-rc3

### LINKED ISSUE #21: 0.10.1-rc3
A newer version is available. Can the webjar be updated?

### PR DESCRIPTION
**Pull request by ilanbiala**
Fixes #21

### CODE REVIEW

**Comment by ilanbiala** [comment]
@jamesward is the change made in this PR correct?

File: pom.xml [hunk 1]
```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>angular-material</artifactId>
-    <version>0.10.1-rc2-SNAPSHOT</version>
+    <version>0.10.1-rc3</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `0.10.1-rc3-SNAPSHOT` since Maven does the actual release.

**Comment by jamesward** [comment]
@ilanbiala I'm not seeing a change to this.

**Comment by ilanbiala** [comment]
@jamesward looks good now?

**Pull request merged**